### PR TITLE
INT: add intention to introduce local variable

### PR DESCRIPTION
**Pull request by Kobzol**
This PR makes the `Introduce variable` refactoring available as an action.

Umbrella issue: https://github.com/intellij-rust/intellij-rust/issues/4926
Fixes: https://github.com/intellij-rust/intellij-rust/issues/2506

### CODE REVIEW

**Comment by vlad20012** [comment]
:+1:

bors r+

File: src/main/resources/intentionDescriptions/IntroduceLocalVariableIntention/before.rs.template [hunk 1]
```diff
@@ -0,0 +1,3 @@
+fn main() {
+    5 + 5;
```
Comment (Undin) [inline src/main/resources/intentionDescriptions/IntroduceLocalVariableIntention/before.rs.template:2]: Maybe surround with `<spot>` attribute?

Comment (Kobzol) [reply src/main/resources/intentionDescriptions/IntroduceLocalVariableIntention/before.rs.template:2]: Added, thanks.

**Comment by Kobzol** [comment]
bors retry

**Pull request merged**